src/client/app/strings/index.ts

```diff
@@ -50,7 +50,17 @@ export function Strings(): ClientStrings {
 }
 
 export function getDisplayLanguage() {
-    return (getRawDisplayLanguage() || 'en').toLowerCase();
+    let rawDisplayLanguage = getRawDisplayLanguage();
```

Comment:
Why not `let rawDisplayLanguage = (getRawDisplayLanguage() || 'en').toLowerCase()`?  And that way you can avoid the if statement and the separate "toLowerCase".  You can even make it `const`, and not need to re-assign the variable...